flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/observer/deployment/AbstractDeploymentObserver.java

```diff
@@ -112,6 +125,7 @@ protected void observeJmDeployment(
         if (JobManagerDeploymentStatus.DEPLOYED_NOT_READY == previousJmStatus) {
             logger.info("JobManager deployment is ready");
             deploymentStatus.setJobManagerDeploymentStatus(JobManagerDeploymentStatus.READY);
+            observeClusterInfo(flinkApp, effectiveConfig);
```

Comment:
`JobManagerDeploymentStatus.READY` does not mean the JobManager Rest API is ready for serving. So we might failed to get the cluster info.

Reply:
Thanks for the suggestion. Updated the logic again, @wangyang0918. PTAL

Reply:
Just noticed the broken e2e tests, looking...

Reply:
v1.14, v1.15 worked fine locally, seemingly 1.13 is broken, I couldn't verify it locally due to missing arm64 based image. I hoped for a pass through..argh...